package.json

```diff
@@ -35,9 +35,10 @@
   "dependencies": {
     "@babel/plugin-proposal-class-properties": "^7.3.4",
     "@babel/plugin-transform-runtime": "^7.3.4",
-    "clean-webpack-plugin": "^2.0.0",
     "@babel/plugin-proposal-object-rest-spread": "^7.3.4",
     "@babel/runtime": "^7.3.4",
+    "bootstrap": "^4.3.1",
```

Comment:
I thought we are using the bootstrap CDN instead of the npm package

Reply:
removed